merino/middleware/logging.py

```diff
@@ -15,26 +16,44 @@ class LoggingMiddleware(BaseHTTPMiddleware):
 
     async def dispatch(self, request: Request, call_next):
         response = await call_next(request)
-        data = {
-            "agent": request.headers["User-Agent"],
-            "path": request.url.path,
-            "method": request.method,
-            "lang": request.headers.get("Accept-Language"),
-            "querystring": dict(request.query_params),
-            "errno": 0,
-            "code": response.status_code,
-            "time": datetime.fromtimestamp(time.time()).isoformat(),
-            # Provided by the asgi-correlation-id middleware.
-            "rid": response.headers["X-Request-ID"],
-            "session_id": request.query_params.get("sid"),
-            "sequence_no": int(seq)
-            if (seq := request.query_params.get("seq"))
-            else None,
-            "country": request.state.location.country,
-            "region": request.state.location.region,
-            "city": request.state.location.city,
-            "dma": request.state.location.dma,
-        }
-        logger.info("", extra=data)
+        if request.url.path.endswith("/api/v1/suggest"):
```

Comment:
To free us from coming back to here again when bumping up the API version in the future, let's do a regex matching here instead:

 ```suggestion
    import re
    from typing imoprt Pattern
    
    # The path pattern for the suggest API 
    PATTERN: Pattern = re.compile(r"/api/v[1-9][0-9]*/suggest$")
 
    if PATTERN.match(request.url.path):
         ...
```